packages/hdwallet-ledger-webusb/src/transport.ts

```diff
@@ -125,10 +154,12 @@ export class LedgerWebUsbTransport extends ledger.LedgerTransport {
     );
 
     try {
-      const methodInstance: LedgerTransportMethod<T, U> = translateCoinAndMethod(this.transport, coin, method);
+      const transport = await getLedgerTransport();
+      const methodInstance: LedgerTransportMethod<T, U> = await translateCoinAndMethod(transport, coin, method);
```

Comment:
Note how this now passes JIT transport vs. `this.transport`, if there's guts of this PR fixing transport, that would be it